learning_resources_search/constants.py

```diff
@@ -135,13 +136,15 @@ class FilterConfig:
         "properties": {
             "id": {"type": "long"},
             "name": {"type": "keyword"},
+            "channel_url": {"type": "keyword"},
         },
     },
     "offered_by": {
         "type": "nested",
         "properties": {
             "code": {"type": "keyword"},
             "name": {"type": "keyword"},
+            "channel_url": {"type": "keyword"},
```

Comment:
Currently the search api returns `channel_url=null ` for the department nested inside course_numbers add `"channel_url": {"type": "keyword"}` there too so that it returns the correct url